doc/dbus/org.opensuse.Agama.Users1.doc.xml

```diff
@@ -54,25 +28,7 @@
     <method name="Write">
       <arg name="result" direction="out" type="u"/>
     </method>
-
-    <!--
-        RootPasswordSet:
-        whenever root password will be set by installer
-    -->
-    <property type="b" name="RootPasswordSet" access="read"/>
-
-    <!--
-        RootSSHKey:
-        Root public ssh key that can be used to login to machine.
-        Can be empty which means not set
-    -->
-    <property type="s" name="RootSSHKey" access="read"/>
-
-    <!--
-        FirstUser:
-        struct( string FullName, string UserName, string Password, boolean AutoLogin, map AdditionalData)
-        Info about first user to set. if Username is empty, it means not set and other values can be ignored
-    -->
-    <property type="(sssba{sv})" name="FirstUser" access="read"/>
+    <property type="(sbs)" name="RootUser" access="read"/>
+    <property type="(sssbba{sv})" name="FirstUser" access="read"/>
```

Comment:
What's the second boolean?
name(s), username(s), password(s), hashedPassword(b), ?????(b), data(a{sv})